.gitignore

```diff
@@ -1,3 +1,6 @@
+# video samples
+samples/
```

Comment:
See comment about creating frames in an ignored folder. Another solution is having the script enforce a folder like "frames" or something.